### ISSUE TITLE
[WebToolsE2E][Aspire] The Aspire dashboard Console log displays duplicate log entries when click the Console button multiple times.

### ISSUE THREAD

**Issue by Rita003** [dotnet/aspire#5102]
**INSTALL STEPS**
1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 8.0.303
3. Install Aspire 8.1.0

**REPRO STEPS**
1. Open CMD, create an Aspire Starter App using following commands:
```
dotnet new aspire-starter -o AspireStarterApp
cd AspireStarterApp/AspireStarterApp.AppHost
```
2. Dotnet run the project in CMD, and open URL in the browser.
`dotnet run`
3. Click the "Console" option on the Aspire dashboard left, and select the apiservice/webfrontend resource.
4. Click the Console button multiple times

**Note:**
1. This issue also repro on Aspire 8.1.1-preview.1.24376.1.

**ACTUAL**
The Aspire dashboard Console logs displays duplicate log entries when click the Console button multiple times.
![Console2](https://github.com/user-attachments/assets/3f2c20f7-eca7-4269-a886-4a5a883b67c9)
![Screenshot 2024-07-29 161008](https://github.com/user-attachments/assets/fe7bddf1-20d8-402e-bd79-fbe09d8eb808)

**EXPECTED**
Log entries not duplicated in the Aspire dashboard Console logs when click the Console button multiple times.
![Screenshot 2024-07-29 160849](https://github.com/user-attachments/assets/3c8f646b-ae14-48f7-81e6-668299968ea0)

**Comment by leslierichardson95** [comment]
Similar to this issue: https://github.com/dotnet/aspire/issues/4213

**Comment by Rita003** [comment]
Verify this issue fixed on the latest Aspire 9.0.0-preview.4.24467.9
